sdk/python/packages/flet-core/src/flet_core/container.py

```diff
@@ -35,6 +38,36 @@
     from typing_extensions import Literal
 
 
+class BlurTileMode(Enum):
+    CLAMP = "clamp"
+    DECAL = "decal"
+    MIRROR = "mirror"
+    REPEATED = "repeated"
+
+
+class ShadowBlurStyle(Enum):
+    NORMAL = "normal"
+    SOLID = "solid"
+    OUTER = "outer"
+    INNER = "inner"
+
+
+@dataclasses.dataclass
+class Blur:
+    sigma_x: float
+    sigma_y: float
+    tile_mode: BlurTileMode = field(default=BlurTileMode.CLAMP)
+
+
+@dataclasses.dataclass
+class BoxShadow:
+    spread_radius: Optional[float] = field(default=None)
+    blur_radius: Optional[float] = field(default=None)
+    color: Optional[str] = field(default=None)
+    offset: OffsetValue = field(default=None)
+    tile_mode: ShadowBlurStyle = field(default=ShadowBlurStyle.NORMAL)
```

Comment:
Inconsistent naming here. `blur_style` could be a better and consistent name. 
What do you think?

Reply:
Good catch, thank you! Will fix that shortly!